### PR TITLE
feat(data): add sub dataset module to the monorepo

### PR DESCRIPTION
**Pull request by douglasduteil**
---

Note "packages/code-du-travail-data/dataset/fiches_service_public/package.json" is ignored due to yarn install failure...

---

**BREAKING CHANGE:** add sub dataset module to the monorepo

This might impact the build script as the newest in range dependencies are used now.

### CODE REVIEW

File: packages/code-du-travail-data/dataset/code_du_travail/eposeidon_script/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "build_eposeidon",
+  "name": "@cdt/data...build_eposeidon",
```
Comment (lionelB) [inline packages/code-du-travail-data/dataset/code_du_travail/eposeidon_script/package.json:2]: why there is `...` in packages name ?

Comment (douglasduteil) [reply packages/code-du-travail-data/dataset/code_du_travail/eposeidon_script/package.json:2]: why not :) 

- `/` si prohibited so we need to find something else as separator
- `-` and `_` are too common

Comment (douglasduteil) [reply packages/code-du-travail-data/dataset/code_du_travail/eposeidon_script/package.json:2]: I made an article about `...` back then :)
https://medium.com/@douglasduteil/npm-scoping-for-free-f162c1db4638

Comment (lionelB) [reply packages/code-du-travail-data/dataset/code_du_travail/eposeidon_script/package.json:2]: > - and _ are too common
😭 I like - and _ 

On a more general note I thought you would move those packages to the monorepo root.

Comment (douglasduteil) [reply packages/code-du-travail-data/dataset/code_du_travail/eposeidon_script/package.json:2]: Maybe under an `@cdt/dataset` namespace 
however moving the files will alter the Python indexation script.
@revolunet do we have kind of Python tests that can help us knowing if the indexation script still work after moving the dataset files ?

Comment (revolunet) [reply packages/code-du-travail-data/dataset/code_du_travail/eposeidon_script/package.json:2]: the create_indexes script itself should fail when it cannot find source files

**Review by lionelB — APPROVED** [review]

**Pull request merged**